### PR TITLE
Remove ElasticSearch port variable and update references in documenta…

### PR DESCRIPTION
**Pull request by rsevilla87**
…tion.

With this change, the ElasticSearch URL should be passed through the variable elasticsearch.url and prometheus.es_url.
Further changes will be required in [benchmark-wrapper](https://github.com/cloud-bulldozer/benchmark-wrapper/pull/219), [e2e-benchmarking](https://github.com/cloud-bulldozer/e2e-benchmarking/pull/61)

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,8 +50,7 @@ metadata:
   namespace: my-ripsaw
 spec:
   elasticsearch:
-    server: "my-es.foo.bar"
-    port: 9200
+    server: "http://my-es.foo.bar:80"
```
Comment (dry923) [inline README.md:53]: Should server be renamed to url?

File: resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml [hunk 2]
```diff
@@ -66,7 +62,7 @@ spec:
                     default: false
                     type: boolean
                   image:
-                    default: "quay.io/cloud-bulldozer/backpack:latest"
+                    default: "quay.io/rsevilla/backpack:latest"
```
Comment (dry923) [inline resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml:65]: Remove before merge ;-)

File: resources/crds/ripsaw_v1alpha1_vegeta_cr.yaml [hunk 3]
```diff
@@ -4,12 +4,12 @@ metadata:
...
-#   server: <es_instance>
-#   port: 9200
-#   index_name: ripsaw-vegeta
+  elasticsearch:
+    # Elastic search instance with full URL format. https://elastic.apps.org:9200
+    server: "https://elastic.apps.org:9200"
+    index_name: ripsaw-vegeta
 # test_user: username_to_attach_to_metadata
-  clustername: myk8scluster
+#  clustername: myk8scluster
```
Comment (dry923) [inline resources/crds/ripsaw_v1alpha1_vegeta_cr.yaml:12]: Why comment out the clustername?

File: resources/crds/ripsaw_v1alpha1_ycsb_cr.yaml [hunk 4]
```diff
@@ -4,6 +4,9 @@ metadata:
   name: ycsb-couchbase-benchmark
   namespace: my-ripsaw
 spec:
+  elasticsearch:
+    server: http://my.elasticsearch.server:80
+    index_name: ripsaw-ycsb
```
Comment (dry923) [inline resources/crds/ripsaw_v1alpha1_ycsb_cr.yaml:9]: Why add this here? it is defaulted to the same? We should standardize the crs w/ or w/o  them

File: README.md (hunk 1 above)
Comment (rsevilla87) [reply README.md:53]: yeah, I like it

File: resources/crds/ripsaw_v1alpha1_ycsb_cr.yaml (hunk 4 above)
Comment (rsevilla87) [reply resources/crds/ripsaw_v1alpha1_ycsb_cr.yaml]: Indeed, I added it to standarize examples, also I removed index_name, which is not present in other examples

File: resources/crds/ripsaw_v1alpha1_vegeta_cr.yaml (hunk 3 above)
Comment (rsevilla87) [reply resources/crds/ripsaw_v1alpha1_vegeta_cr.yaml]: fixed

File: docs/elastic.md [hunk 5]
```diff
@@ -36,8 +41,7 @@ spec:
   test_user: homer_simpson
   clustername: test_ci
   elasticsearch:
-    server: my.es.server
-    port: 8080
+    url: "http://my.es.server:9200"
```
Comment (dry923) [inline docs/elastic.md:44]: Can we give an example with a user/pass being passed as well?

File: docs/metadata.md [hunk 6]
```diff
@@ -36,7 +36,6 @@ the workload template with an init container section that looks like:
         command: ["python3", "stockpile-wrapper.py"]
         args:
           - -s={{ elasticsearch.server }}
-          - -p={{ elasticsearch.port }}
```
Comment (dry923) [inline docs/metadata.md:39]: Elastic search.server needs to be .url ^

File: docs/elastic.md (hunk 5 above)
Comment (rsevilla87) [reply docs/elastic.md:44]: done!

File: docs/metadata.md (hunk 6 above)
Comment (rsevilla87) [reply docs/metadata.md:39]: fixed!

**Comment by dry923** [comment]
@rsevilla87 remind me. these are "failed" because of the vars we pass in jenkins correct? Assuming so, can we figure out a time to flip it so you can get a happy green check mark?

**Comment by rsevilla87** [comment]
> @rsevilla87 remind me. these are "failed" because of the vars we pass in jenkins correct? Assuming so, can we figure out a time to flip it so you can get a happy green check mark?

This one is depending on https://github.com/cloud-bulldozer/benchmark-wrapper/pull/219 (which passed tests after changing the ES url in the jenkins job)
e2e-benchmark scripts also will need this PR https://github.com/cloud-bulldozer/e2e-benchmarking/pull/61

**Review by jtaleric — APPROVED** [review]
LGTM

**Pull request merged**